src/h5cpp/datatype/ebool.cpp

```diff
@@ -29,6 +29,7 @@
 #include <h5cpp/datatype/ebool.hpp>
 #include <h5cpp/error/error.hpp>
 
+#ifndef _MSC_VER
```

Comment:
Why is it commented for MCS?

Reply:
If I recall correctly this was an issue with the MS compiler and enumerations which popped up when using Catch2. That's one of the reasons I guess why I've created #503 to get rid of all those problems. It was a mistake (mainly by me) to allow all-caps enumerations in our code.

Reply:
I hope I can fix the Visual Studio problems tomorrow. However, I have no glue what's wrong with Centos 7.